src/components/hooks/use-client-country.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
+import { cloudflare_trace_url } from 'common/constants'
+
+export const useClientCountry = () => {
+    const [clients_country, setClientsCountry] = useState('')
+
+    useEffect(() => {
+        const fetchClientCountry = async () => {
```

Comment:
Hi @yashim-deriv , do you think, we need some kind of fallback here? In case the API fails for some kind limit or other reason?